end-user-surveys/analysis-guides/introduction.md

```diff
@@ -0,0 +1,49 @@
+
+The goal is not to enforce a rigid process, but to make survey analysis reproducible, transparent, and consistent across SIGs.
+
+## Who this Tutorial is For
+This tutorial is designed for a broad audience within the OTel community, including:
+
+* **SIG Leads** who want to better understand community feedback to guide their working group’s direction.
+* **Contributors** who may be helping with survey design or analysis.
+* **Community Managers** who communicate findings back to the community.
+* **Analysts and Researchers** who want to ensure their work aligns with the shared OTel methodology.
+
+No prior experience in data science or statistics is required. The tutorial provides plain-language explanations, practical examples, and tool-agnostic workflows that can be applied using Google Spreadsheets, Python, SQL, or any preferred toolset. But for this tutorial, **Google Spreadsheets** will be used for sample analysis.
+
+## What You’ll Learn
+By following this tutorial, you’ll learn how to:
+* Plan your analysis from the start by drafting survey questions with analysis in mind.
+* Clean and structure raw survey data to prepare it for analysis and exploration.
+* Analyze data systematically, using descriptive statistics and (optional) inferential techniques.
+* Visualize key insights effectively, with emphasis on clear storytelling.
+* Document and communicate your findings, ensuring reproducibility for future SIGs.
+
+Throughout the tutorial, we’ll use screenshots and examples to show exactly what each step looks like in practice.
+
+## Data Source and Format
+All survey data in this tutorial comes from Google Forms, which automatically stores responses in Google Sheets. This makes Google Sheets the primary workspace for both data storage and analysis, since it’s easily accessible to anyone across SIGs without requiring additional setup.
+
+Each exported survey sheet typically contains:
+* One row per respondent, with a unique timestamp.
+* One column per survey question, using the exact question text as the column header.
+* Multiple-choice or multi-select questions represented as text values (e.g., “Kubernetes”), sometimes separated by commas if respondents selected multiple options.
+* Open-ended questions are represented as free text.
+
+Before starting any analysis, it’s important to familiarize yourself with this structure so you know where to find demographic questions, key variables, and open-ended responses.
+
+** Insert Image here
+
+
+**Tip**: ***Make a copy of the original Google Sheet and work from the copy. This preserves the raw data in case you need to revert or share it with others later.*** 
+
```

Comment:
I think this would fit better to the file end-user-surveys/analysis-guides/survey-analysis.md.